riptide-spring-boot-autoconfigure/README.md

```diff
@@ -154,15 +154,15 @@ Required when `logging` is enabled.
 </dependency>
 ```
 
-#### [Tracer](https://github.com/zalando/tracer) integration
+#### [OpenTracing Toolbox](https://github.com/zalando/opentracing-toolbox) integration
```

Comment:
A few paragraphs down is a `Tracing` section, what about this one? It's a bit confusing as both are saying "required when tracing is enabled"

Reply:
Re-arranged and reworded. Please take another look.